pandas/core/arrays/base.py

```diff
@@ -91,21 +91,6 @@
     nargsort,
 )
 
-if TYPE_CHECKING:
-
-    class ExtensionArraySupportsAnyAll("ExtensionArray"):
-        def any(self, *, skipna: bool = True) -> bool:
-            pass
-
-        def all(self, *, skipna: bool = True) -> bool:
-            pass
-
-    from pandas._typing import (
-        NumpySorter,
-        NumpyValueArrayLike,
-    )
```

Comment:
sorry, I meant to only move `class ExtensionArraySupportsAnyAll`

So here you'd still have
```
if TYPE_CHECKING:
    from pandas._typing import (
        NumpySorter,
        NumpyValueArrayLike,
    )
```

and below `class ExtensionArray` you'd have `class ExtensionArraySupportsAnyAll`

Reply:
Ah okay sure thing I'll move that now.